asterisk/rootfs/usr/share/tempio/pjsip_default.conf.gtpl

```diff
@@ -15,31 +15,30 @@ password={{ $secret }}
 ; Common AOR parameters (template)
 [sipjs-phone-aor](!)
 type=aor
-max_contacts=5
+max_contacts=6
 remove_existing=yes
+remove_unavailable=yes
 
 ; Common ENDPOINT parameters (template)
 [sipjs-phone-endpoint](!) 
 type=endpoint
 send_rpid=yes
 send_pai=yes
 device_state_busy_at=1
-dtls_auto_generate_cert=yes
 webrtc=yes
 ; Setting webrtc=yes is a shortcut for setting the following options:
 ; use_avpf=yes
 ; media_encryption=dtls
+; dtls_auto_generate_cert=yes (if dtls_cert_file is not set)
```

Comment:
wow this one is nice!

Reply:
Yes, see config docs here: https://wiki.asterisk.org/wiki/display/AST/Asterisk+18+Configuration_res_pjsip#Asterisk18Configuration_res_pjsip-endpoint_dtls_auto_generate_cert